src/plugins/memdump/memdump.cpp

```diff
@@ -111,7 +111,45 @@
 
 #define DUMP_NAME_PLACEHOLDER "(not configured)"
 
-static bool dump_memory_region(drakvuf_t drakvuf, vmi_instance_t vmi, drakvuf_trap_info_t* info, memdump* plugin, access_context_t* ctx, size_t num_pages, const char* reason)
+// TODO move to common library, this is also used in procmon
+static vmi_pid_t get_pid_from_handle(memdump* f, drakvuf_t drakvuf, drakvuf_trap_info_t* info, addr_t handle)
```

Comment:
I propose to move this to libdrakvuf, as such method already exists in procmon plugin, but it's actually a general-purpose method.